edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/aop/CourseManagementAdministrationAuthzAdvisor.java

```diff
@@ -42,7 +43,8 @@ public void before(Method method, Object[] oa, Object obj) throws Throwable {
 
 		// We can't check the standard site- or group- or resource-based authorization for modifying CM data,
 		// since CM isn't scoped by sakai references.  So we allow only the super user.
-		if(!securityService.isSuperUser()) {
+		if(!securityService.isSuperUser() &&
```

Comment:
is the isSuperUser check still needed as a security check will always allow admin user?

Reply:
Good point. I updated to remove the isSuperUser() check.